server/src/test/resources/config/sidecar_single_instance_non_zero_port.yaml

```diff
@@ -16,7 +16,7 @@ cassandra:
 
 sidecar:
   host: 0.0.0.0
-  port: 9043
+  port: 9046
```

Comment:
because we have unit tests running in parallel, I saw collisions between the `updateTrafficShapingOptionsWithNonZeroListenPort` and the new tests in `CassandraSidecarDaemonTest` where both were trying to use the same port. So I changed the `updateTrafficShapingOptionsWithNonZeroListenPort` test to use any non-zero port that doesn't collide with port 9043